### PR TITLE
Add Atom support

### LINKED ISSUE #5: Add Atom support
Add support for Github’s Atom editor.

### PR DESCRIPTION
**Pull request by Rexogamer**
**Closes #5**

This PR adds support for opening the newly created repo in Atom.

### CODE REVIEW

File: automate-project.py [hunk 1]
```diff
@@ -132,12 +132,19 @@ def CreateGitHubRepo():
...
 
 
-# opens project in VS code
-if config.get("DEFAULT", "vscode") == "on":
+# opens project in editor
+if config.get("DEFAULT", "editor") == "vscode":
     subprocess.run("code .", shell=True)
+elif config.get("DEFAULT", "editor" == "atom":
+    subprocess.run("atom .", shell=True)
+elif config.get("DEFAULT", "editor" == "none":
```
Comment (jarodburchill) [inline automate-project.py:140]: Nice touch!

File: README.md [hunk 2]
```diff
@@ -77,5 +79,6 @@ react-ts
 ## Contributors:
 <a href="https://github.com/jarodburchill"><img src="https://avatars.githubusercontent.com/u/37840393?v=3" title="jarodburchill" width="80" height="80"></a>
 <a href="https://github.com/ajnieset"><img src="https://avatars.githubusercontent.com/u/40476295?v=3" title="ajnieset" width="80" height="80"></a>
+<a href="https://github.com/rexogamer"><img src="https://avatars.githubusercontent.com/u/42586271?v=3" title="rexogamer" width="80" height="80"></a>
```
Comment (jarodburchill) [inline README.md:82]: Thanks for doing this for me!

**Review by jarodburchill — COMMENTED** [review]
Everything looks good to me, Thanks!

File: automate-project.py (hunk 1 above)
Comment (Rexogamer) [reply automate-project.py:140]: Thanks!

File: README.md (hunk 2 above)
Comment (Rexogamer) [reply README.md:82]: No problemo!

**Pull request merged**